src/components/PlayerNotesModal/PlayerNotesModal.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2012-2021  Online-Go.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as React from "react";
+import {_} from 'translate';
+import {Modal, openModal} from "Modal";
+import { get, set } from "remote_storage";
+declare let swal;
+
+interface Events {
+}
+
+interface PlayerNotesModalProperties {
+    playerId: number;
+}
+
+export class PlayerNotesModal extends Modal<Events, PlayerNotesModalProperties, any> {
+
+    playerNotes = {};
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            notes: undefined
+        };
+    }
+
+    componentDidMount = () => {
+        super.componentDidMount(); /* this.close() doesn't work if you don't do this */
+        get("player-notes")
```

Comment:
@anoek I just realized the interface for `remote_storage` and `local_storage` are the same.  Is this desirable, given that the status quo for pulling in modules is `import {x} from 'y'`?  Hard to tell what is being done without scrolling up to the imports...

Reply:
Could probably use some pondering and improvement, currently, for no good reason other than "that's how it's always been", `data`, `preferences`, and now `remote_storage` all use a `import * as data from "data"` style import, whereas everything else uses `import { x } from 'x'` style, except `ITC` which uses `import ITC from "ITC"` style.

We should probably normalize those a bit, possibly grouping and refactoring the functionality along the way. Currently we have `data`, `preferences`, `remote_data`, and `ITC`, all of which are used for managing persistent state and synchronization in different ways. It seems like maybe we could refactor that code into a tidier state management library or something. But that's it's own can of worms..

Reply:
Oh of course, I forgot that `data` gets pulled in like that!  In that case, it probably makes sense to do the same with `remote_storage`.